share/spice/tor_node/tor_node.js

```diff
@@ -115,72 +200,13 @@
         return base;
     }
 
-    // Data that's shared between the two Spice.add calls.
-    function get_shared_spice_data(api_result) {
-        // Filter out the trigger from the query.
-        var words = DDG.get_query().split(" ");
-        var query = (words[0] =~ /tor|onion/i) ? words.pop() : words.shift();
-
-        // Collect together and classify the nodes.
-        var i, node, nodes = [];
-        for (i = 0; i < api_result.relays.length; i++) {
-            node = api_result.relays[i];
-            node.ref_name = "Fingerprint";
-            node.ref = node.fingerprint;
-            node.type = "Relay";
-            nodes.push(node);
-        }
-        for (i = 0; i < api_result.bridges.length; i++) {
-            node = api_result.bridges[i];
-            node.ref_name = "Hashed Fingerprint";
-            node.ref = node.hashed_fingerprint;
-            node.type = "Bridge";
-            nodes.push(node);
-        }
-        if (nodes.length === 0) {
-            return null;
-        }
-
-        for (i = 0; i < nodes.length; i++) {
-            node = nodes[i];
-
-            if (node.nickname && node.nickname !== "Unnamed") {
-                node.title = node.nickname;
-            }
-            else {
-                node.title = "Unnamed " + node.type;
-            }
-
-            if (node.advertised_bandwidth) {
-                node.bandwidth = prettifyBitrate(node.advertised_bandwidth);
-            }
-
-            node.url = "https://atlas.torproject.org/#details/" + node.ref;
-        }
-
-        return {
-            id: "tor_node",
-            name: "Tor Node",
-            nodes: nodes,
-            meta: {
-                sourceName: "Tor Atlas",
-                sourceUrl: "https://atlas.torproject.org/#search/" + query
-            },
-            templates: {
-                options:{
-                    moreAt: true
-                }
-            }
-        };
-    };
-
     function stripPort(addr) {
         return addr.replace(/:[0-9]+$/, "");
```

Comment:
This function needs to be updated to remove the brackets from an IPv6 address, these are not required if the [port number is removed](http://en.wikipedia.org/wiki/IPv6_address#Literal_IPv6_addresses_in_network_resource_identifiers)

Example IPv6:

Before: `[2a01:4f8:100:112c::57]:9001`
After: `2a01:4f8:100:112c::57`